src/libstd/net/ip.rs

```diff
@@ -244,6 +243,20 @@ impl FromInner<libc::in_addr> for Ipv4Addr {
     }
 }
 
+impl Into<u32> for Ipv4Addr {
+    fn into(self) -> u32 {
+        let ip = self.octets();
+        (ip[0] as u32) << 24 + (ip[1] as u32) << 16 + (ip[2] as u32) << 8 + (ip[3] as u32)
+    }
+}
+
+impl From<u32> for Ipv4Addr {
```

Comment:
Can you mark this and the above impl block with `#[stable(feature = "ip_u32", since = "1.1.0")]` as well?